### ISSUE TITLE
PR that added session_affinity variable to serverless_neg was incomplete

### ISSUE THREAD

**Issue by uap-universe** [terraform-google-modules/terraform-google-lb-http#332]
### TL;DR

The PR #319 added a lookup for the `session_affinity` key, but - as can be seen in the code below - did not add the key in the `variables.tf`.

https://github.com/terraform-google-modules/terraform-google-lb-http/blob/a0ea2acb64ab64a130b7aced40aa25644924499b/modules/serverless_negs/variables.tf#L60-L71

Compare with `dynamic_backends` line 97:

https://github.com/terraform-google-modules/terraform-google-lb-http/blob/a0ea2acb64ab64a130b7aced40aa25644924499b/modules/dynamic_backends/variables.tf#L83-L100

### Expected behavior

Map keys that are looked up in `main.tf` are defined in `variables.tf`.

### Observed behavior

The map key `session_affinity` is looked up in `main.tf` but **not** defined in `variables.tf`.

### Terraform Configuration

```hcl
irrelevant - see code
```

### Terraform Version

```sh
irrelevant - see code
```

**Comment by imrannayer** [comment]
Need to add `connection_draining_timeout_sec`, `session_affinity` and `affinity_cookie_ttl_sec` in Serverless NEG's variables.tf. Change is needed in [template file](https://github.com/terraform-google-modules/terraform-google-lb-http/blob/master/autogen/variables.tf.tmpl)
